code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -40,7 +40,7 @@
 	M.do_attack_animation(src)
 	if(!M.environment_smash)
 		return
-	if(M.environment_smash == 3)
+	if(M.environment_smash == ENVIRONMENT_SMASH_RWALLS)
```

Comment:
Should this be changed to >= just in case another define is added in the future?